src/python_testing/TC_DRLK_2_13.py

```diff
@@ -0,0 +1,663 @@
+        self.alirocredentialissuerkey_secondary = bytes.fromhex(
+            "047a4c112d753924cdf3779a3c84fec2debaa6f0b3084450878acc7ddcce7856ae57b1ebbe2561015103dd7474c2a183675378ec55f1e465ac3436bf3dd5ca54d4")
+        self.aliroevictableendpointkey = bytes.fromhex(
+            "047a4c772d753924cdf3779a3c84fec2debaa6f0b3084450878acc7ddcce7856ae57b1ebbe2561015103dd7474c2a183675378ec55f1e465ac3436bf3dd5ca54d4")
+        self.alirononevictableendpointkey = bytes.fromhex(
+            "047a4c662d753924cdf3779a3c84fec2debaa6f0b3084450878acc7ddcce7856ae57b1ebbe2561015103dd7474c2a183675378ec55f1e465ac3436bf3dd5ca54d4")
+        self.alirononevictableendpointkey1 = bytes.fromhex(
+            "047a4c552d753924cdf3779a3c84fec2debaa6f0b3084450878acc7ddcce7856ae57b1ebbe2561015103dd7474c2a183675378ec55f1e465ac3436bf3dd5ca54d4")
+        #  step 1 TH reads DUT Endpoint 0 OperationalCredentials cluster CurrentFabricIndex attribute
+        self.step("1a")
+        self.fabric_idx1 = await self.read_attributes_from_dut(endpoint=self.common_cluster_endpoint,
+                                                               cluster=Clusters.Objects.OperationalCredentials,
+                                                               attribute=Clusters.OperationalCredentials.Attributes.CurrentFabricIndex
+                                                               )
+        self.step("1b")
+        if self.pics_guard(self.check_pics("DRLK.S.C1d.Rsp")):
+            await self.send_clear_user_cmd(user_index=int(0xFFFE))
+
+        self.step("1c")
+        if self.pics_guard(self.check_pics("DRLK.S.C26.Rsp")):
+            await self.clear_credentials_cmd(credential=NullValue)
+
+        # step 2
+        self.step("2a")
+        if self.pics_guard(self.check_pics("DRLK.S.C26.Rsp")):
+            await self.send_clear_aliro_reader_config_cmd()
+        self.step("2b")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=False, expected_status=Status.Success)
+        self.step("2c")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=True, expected_status=Status.Success)
+        # step 3,4,5
+        aliro_attribute_verify_steps = [
+            {"th_step": "3", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroReaderVerificationKey,
+             "aliro_attribute_value": self.verificationKey},
+            {"th_step": "4", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroReaderGroupIdentifier,
+             "aliro_attribute_value": self.groupIdentifier},
+            {"th_step": "5", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroGroupResolvingKey,
+             "aliro_attribute_value": self.groupResolvingKey}
+        ]
+        list_of_verification_objects = [AliroAttributeVerify(**aliro_object) for aliro_object in
```

Comment:
create the objects in the list, do not convert from dict.

Reply:
dict converter has been removed